tslearn/docs/user_guide/dtw.rst

```diff
@@ -51,7 +51,7 @@ The following image exhibits the DTW path (in white) for a given pair of time
 series, on top of the cross-similarity matrix that stores :math:`d(x_i, y_j)`
 values.
 
-.. figure:: ../_static/img/dtw_thumb.png
+.. figure:: ../_images/sphx_glr_plot_dtw_thumb.svg
```

Comment:
Link fails

Reply:
Yeah, I already found it weird that this was the only one with just '../' as prefix while the others have '../../'! Let's hope the fix works :)